src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+
+  const noSpacesArray = sourceString.split(';');
+  const arrayOfArrays = noSpacesArray
+    .map((element) => element.split(':'))
+    .filter((element) => element.length === 2)
```

Comment:
The filter operation here is good for ensuring that only valid declarations are processed, but you can combine the map and filter operations into a single map operation to streamline the code. You can filter out invalid declarations within the map by checking the length of the split array before trimming and adding it to the object.